Source/WebKit/GPUProcess/webrtc/RemoteSampleBufferDisplayLayer.cpp

```diff
@@ -29,7 +29,7 @@
 #if PLATFORM(COCOA) && ENABLE(GPU_PROCESS) && ENABLE(MEDIA_STREAM)
 
 #include "GPUConnectionToWebProcess.h"
-#include "IPCTester.h"
+#include "IPCUtilities.h"
```

Comment:
Shouldn't the header say something about "Testing" in the name? IPCUtilities doesn't make it clear this is for testing.

Reply:
The initial idea was that most likely there won't be many testing related functions in addition to this. This would mean the "Testing" header would be a bit too specific.
However, there likely is some utilities that could be moved to the header. This would mean that having testing utilities in "Utilities" header would be appropriate.